notebooks/official/experiments/get_started_with_comparing_local_trained_models.ipynb

```diff
@@ -0,0 +1,1027 @@
+{
```

Comment:
<p>Clarify why one is defining a baseline.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/558/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
<p>done.</p>